src/Components/Pages/RentStabilization/RentStabilization.tsx

```diff
@@ -37,7 +49,7 @@ export const RentStabilization: React.FC = () => {
             </p>
           </ContentBoxItem>
         </ContentBox>
-
+        <div className="divider__print" />
```

Comment:
oh I meant to write this in a comment but forgot --
the dividers I added to `RentStabilization.tsx` and `PortfolioSize.tsx` aren't showing up as expected in the print dialog. I was wondering if you might see something that's going on that I wasn't catching @austensen

Reply:
I made sure the style sheets were being imported at the top, tried `!important`, etc.

Reply:
I think it might just need `width: 100%`

Reply:
tysm, that did it!